airbyte-integrations/connectors/source-surveycto/source_surveycto/helpers.py

```diff
@@ -13,6 +13,20 @@
 
 
 class Helpers(object):
+
+    @staticmethod
+    def modify_date(response_json):
+        data_mod = ["startdata", "enddata", "CompletionDate", "SubmissionDate"]
```

Comment:
Shouldn't these be "starttime" and "endtime" instead of "startdata" and "enddata"?

Reply:
done